content/tos/index.md

```diff
@@ -0,0 +1,113 @@
+
+We reserve the right to modify this Agreement prospectively at any time. We will post any updates to this Agreement on tlon.io, and will indicate the date this Agreement was last revised. By continuing to use the Service after the effective date of any modifications to this Agreement, you agree to be bound by the modified terms. It is your responsibility to check tlon.io regularly for modifications to this Agreement. We last modified this Agreement on the date listed at the top of this Agreement.
+
+PLEASE NOTE THAT ANY DISPUTE OR CLAIM RELATED TO THIS AGREEMENT OR ARISING OUT OF YOUR ACCESS TO OR USE OF THE PLATFORM MUST BE RESOLVED BY ARBITRATION ON AN INDIVIDUAL BASIS, AND MAY NOT BE ARBITRATED OR OTHERWISE PURSUED AS A CLASS ACTION. PLEASE SEE SECTION 16 BELOW.
+
+1. Overview of the Service.
+   1. You may access and use the Service in accordance with this Agreement. You will comply with the terms of this Agreement and all laws, rules and regulations applicable to your use of the Service.
+   2. You may use any software, data, text, audio, video or images made available to you by any third party or in conjunction with the Service at your election (“Third-Party Content”). Third-Party Content is governed by this Agreement and, if applicable, separate terms and conditions accompanying such Third-Party Content, which terms and conditions may include separate fees and charges.
+   3. We reserve the right, at any time, to modify, suspend, or discontinue the Service, or any part thereof, upon thirty (30) days’ prior written notice to you, except that this notice will not be required if the thirty (30) day notice period (a) would pose a security or intellectual property issue to us or the Service, (b) is economically or technically burdensome, or (c) would cause us to violate legal requirements. You agree that Tlon will not be liable to you or to any third party for any modification, suspension, or discontinuance of the Service, or any part thereof.
+   4. This Agreement and the Service do not contain any service level obligations whatsoever with respect to the availability of the Service, whether by Tlon or otherwise. By entering into this Agreement, you acknowledge and agree that the Service may not be available to you at all times during the Term (defined below).
+2. Eligibility.
+   1. To access or use the Service, you must have a valid email address and a valid form of payment. You acknowledge and agree that you will only create one account per email address. During the user registration process for the Service, you will be required to agree to this Agreement.
+   2. During the registration process, you will be provided an Urbit ship if you do not already have one. You will be responsible for custody of the ownership keys of your ship. If you lose these, we cannot recover them for you, and you could lose access to your ship and your data permanently.
```

Comment:
We say "Urbit ship" in a couple places, but shouldn't all of those say "Urbit ID" instead? They talk about ownership creds etc, which doesn't factor in to just your ship (ie pier, files) itself.

Reply:
Yes, this is a good change to make globally in the document.